flow/layers/physical_shape_layer.cc

```diff
@@ -62,90 +66,75 @@ void PhysicalShapeLayer::Preroll(PrerollContext* context,
     set_paint_bounds(path_.getBounds());
   } else {
 #if defined(OS_FUCHSIA)
-    // Let the system compositor draw all shadows for us.
```

Comment:
Sorry this diff is so terrible.  I copy-pasted all of this shadow code as-is, I just added an if() else around it

Reply:
Do you have some benchmarks that compare the performance between `kRenderPhysicalShapeUsingSystemCompositor` is `true` and `false`?

Reply:
I focused on redoing the CL for your other comments.  I will work on getting benchmarks now